src/components/ReactFlagsSelect/ReactFlagsSelect.tsx

```diff
@@ -103,6 +107,9 @@ const ReactFlagsSelect: React.FC<Props> = ({
 
     const filteredCountriesOptions = countriesOptions.filter((key) => {
       const label = getLabel(key);
+      if (typeof label === "object") {
+        return label && label.primary.match(new RegExp(value, "i"));
```

Comment:
On a second thought, let's use optional chaining here `label?.primary?.match(`
Please help update the return statement below to use optional chaining too 🙏🏾

Reply:
I think we should also search the secondary value -
`label?.primary?.match(new RegExp(value, "i")) || label?.secondary?.match(new RegExp(value, "i"))`